### PR TITLE
Trying to get pytest working

### PR DESCRIPTION
**Pull request by mzieg**
Discussion in ENLIGHTEN pull request

### CODE REVIEW

File: wasatch/FeatureIdentificationDevice.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def __init__(self, device_id: str, message_queue: list = None) -> None:
         self.message_queue = message_queue
 
         self.device = None
-        if "MOCK" in str(device_id):
+        if "MOCK" in str(device_id).upper():
```
Comment (rc14193) [inline wasatch/FeatureIdentificationDevice.py:93]: Does this need to use .upper? Looking at the other DeviceIDs they're already upper case (FILE, USB), so I'm thinking MOCK should be upper case already and Mock would be an invalid DeviceID.

File: wasatch/WasatchDeviceWrapper.py [hunk 2]
```diff
@@ -249,7 +236,7 @@ def connect(self):
             is_ble         = self.is_ble)
         log.debug("device wrapper: Instance created for worker")
 
-        self.wrapper_worker.setDaemon(True)
+        self.wrapper_worker.daemon = True
```
Comment (mzieg) [inline wasatch/WasatchDeviceWrapper.py:239]: fixed deprecation warning

**Review by rc14193 — APPROVED** [review]

File: wasatch/FeatureIdentificationDevice.py (hunk 1 above)
Comment (mzieg) [reply wasatch/FeatureIdentificationDevice.py:93]: Well, without this change, all release tests now fail.  When I put a debug line in the `else` block, I get this:
```
FID.ctor: device_id was <MockUSBDevice 0x731d4:0x4000:111111:111111> so device_type is RealUSBDevice
```

File: wasatch/MockUSBDevice.py [hunk 3]
```diff
@@ -78,12 +85,13 @@ def __init__(self, spec_name, eeprom_name, eeprom_overrides=None, spectra_option
...
         # if first tier, where wValue matters then wValue should be given as None
+        log.info("MockUSBDevice: creating cmd_dict")
         self.cmd_dict = {
             (0xb2,None): self.cmd_set_int_time,
             (0xb6,None): self.cmd_set_offset,
             (0xb7,None): self.cmd_set_gain,
-            (0xbe,None): self.cmd_toggle_laser,
-            (0xd6,None): self.cmd_toggle_tec,
+            (0xbe,None): self.cmd_set_laser_enable,
+            (0xd6,None): self.cmd_set_detector_tec_enable,
```
[the remark this answers is not in the document]
Comment (mzieg) [reply wasatch/MockUSBDevice.py:93]: To me, and this is very much an opinion, "toggle_laser", "toggle_tec" etc mean "change it to the opposite of what it is now."  Such a function wouldn't take any argument at all.  However, we do pass a boolean flag to these, so to me it means, "set the 'enable' flag to the value I'm sending".

A little more generally, I think in terms of settor and gettor accessors, and if we have a "get_laser_enable", we should have a "set_laser_enable".  I'm not dogmatic about it, but to me that seems the most universally readable API.

**Pull request merged**